builds/wasm/src/vade/typings/index.ts

```diff
@@ -13,88 +13,73 @@ import {
   UnfinishedBbsCredential as UnfinishedCredentialBbs,
   UnsignedCredential as UnsignedCredentialBbs,
 } from './vade-evan-bbs';
+
 import {
-  BigNumber,
-  CreateRevocationRegistryDefinitionResult,
-  Credential as CredentialCl,
-  CredentialDefinition as CredentialDefinitionCl,
-  CredentialOffer as CredentialOfferCl,
-  CredentialPrivateKey,
-  CredentialProposal as CredentialProposalCl,
-  CredentialRequest as CredentialRequestCl,
-  CredentialSchema as CredentialSchemaCl,
-  CredentialSecretsBlindingFactors,
-  EncodedCredentialValue as EncodedCredentialValueCl,
-  IssueCredentialResult as IssueCredentialResultCl,
-  MasterSecret as MasterSecretCl,
-  ProofCredential as ProofCredentialCl,
-  ProofPresentation as ProofPresentationCl,
-  ProofRequest as ProofRequestCl,
-  RevocationIdInformation,
-  SchemaProperty as SchemaPropertyCl,
-  SubProofRequest as SubProofRequestCl,
-} from './vade-evan-cl';
+  AssertionProof as AssertionProofJwt,
+  Credential as CredentialJwt,
+  CredentialSchemaReference as CredentialSchemaReferenceJwt,
+  CredentialStatus as CredentialStatusJwt,
+  CredentialSubject as CredentialSubjectJwt,
+  IssueCredentialPayload as IssueCredentialPayloadJwt,
+  IssueCredentialResult as IssueCredentialResultJwt,
+  ProofVerification as ProofVerificationJwt,
+  SchemaProperty as SchemaPropertyJwt,
+  SignerOptions as SignerOptionsJwt,
+  TypeOptions as TypeOptionsJwt,
+  UnsignedCredential as UnsignedCredentialJwt,
+  VerifyProofPayload as VerifyProofPayloadJwt,
+} from './vade-jwt-vc';
```

Comment:
I think this one is missing, cant run the tests.

Reply:
This is because you havent cloned vade-jwt-vc